.gitignore

```diff
@@ -32,6 +32,6 @@ docs_builder/notebooks/.ipynb_checkpoints/
 misc/biomodels/r31_uncurated/
 
 # documentation build
-docs_builder/_build/
```

Comment:
This has to be reverted. The build files do NOT belong in the repository. These are created on the fly via readthedocs. I.e. revert the changes and remove all the `_docs_build/_build` files from the pull request.